spec/features/create_audio_object_spec.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create a new Audio object via Pre-assembly', type: :feature do
+  audio_druid = '' # used for preassembly manifest files
```

Comment:
Seeing a variable declaration in an RSpec `describe` block throws me a bit. It's perfectly fine Ruby, of course, and RSpec is just Ruby. I'm wondering if perhaps `let!(:audio_druid) { '' }` would work just as well and might be less likely to cause me to blink?

Reply:
I will try this, but I believe the memoized value messes things up;  I had a similar problem with ETD test.

Reply:
confirmed that `let` and `let!` will not work.

Reply:
:hankey: 

thanks for giving it a shot!